fdbclient/NativeAPI.actor.cpp

```diff
@@ -6640,10 +6648,12 @@ ACTOR Future<GetReadVersionReply> getConsistentReadVersion(SpanContext parentSpa
 						    "TransactionDebug", debugID.get().first(), "NativeAPI.getConsistentReadVersion.After");
 					ASSERT(v.version > 0);
 					cx->minAcceptableReadVersion = std::min(cx->minAcceptableReadVersion, v.version);
-					if (cx->isCurrentGrvProxy(v.proxyId)) {
-						cx->ssVersionVectorCache.applyDelta(v.ssVersionVectorDelta);
-					} else {
-						continue; // stale GRV reply, retry
+					if (cx->versionVectorCacheActive(v.ssVersionVectorDelta)) {
+						if (cx->isCurrentGrvProxy(v.proxyId)) {
+							cx->ssVersionVectorCache.applyDelta(v.ssVersionVectorDelta);
+						} else {
+							continue; // stale GRV reply, retry
```

Comment:
Unrelated to this change, this line can potentially become a busy loop. Maybe we should add a backoff here in a separate PR.

Reply:
On "continue" another "GetReadVersionRequest" gets sent, and wouldn't it wait on the "choose" statement after that? Trying to understand how it will become a busy loop. But maybe we can discuss that offline.